.github/workflows/dod.yml

```diff
@@ -1,7 +1,7 @@
 name: Definition of Done
 on:
   pull_request:
-    types: [opened, edited]
+    types: [edited]
```

Comment:
opened is also fine here. Sorry, I was not clear in my comment.

Reply:
But theoretically DoD checks it the checkmarks had been placed in the PR description DoD section, so they shouldn't be present there when the PR had just been opened? Since the only way they can be added is through edits. I'm only not sure about reopened PRs.

Reply:
But DoD action also places those checkmarks when the PR is opened

Reply:
OK then, I left it there anyway, so should all be good now.